### PR TITLE
fix(ui): tabler icons after testing

### PR DESCRIPTION
**Pull request by ksnitsky**
Part Of #60.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/views/ats/candidates/_header_show.html.slim [hunk 1]
```diff
@@ -4,9 +4,9 @@ ruby:
   red_name = @candidate.blacklisted?
 
 - candidate_name = capture
-    h1.fw-bold.mb-0.text-break class=("text-danger" if red_name)
+    h1.mb-0.text-break class=("text-danger hstack gap-2" if red_name)
       - if red_name
-        = render IconComponent.new(:ban, size: :medium, class: "me-2")
+          = render IconComponent.new(:ban, size: :medium)
```
Comment (ksnitsky) [inline app/views/ats/candidates/_header_show.html.slim:9]:
```suggestion
        = render IconComponent.new(:ban, size: :medium)
```

**Comment by ksnitsky** [comment]
Everything except changing the color of the table sorting arrows

**Review by wrmk — COMMENTED** [review]
image
 "Blacklisted icon should be centered" seems work, but "blacklisted looks strange", please create an issue

**Review by wrmk — APPROVED** [review]

**Comment by ksnitsky** [comment]
> image "Blacklisted icon should be centered" seems work, but "blacklisted" looks strange, please create an issue

Fix is in https://github.com/freeats/freeats/pull/42

**Pull request merged**